src/model/document.js

```diff
@@ -391,6 +396,28 @@ export default class Document {
 	 * @event change
 	 * @param {module:engine/model/batch~Batch} batch The batch that was used in the executed changes block.
 	 */
+
+	/**
+	 * Fired after each {@link module:engine/model/model~Model#enqueueChange `enqueueChange()` block},
```

Comment:
You can now update the last example in the `Document#change`'s docs. It's outdated now. These events should be properly cross-linked between each other.

Reply:
I recommend moving most of the documentation to `Document#change` and clarifying there, in one place, which event to use and when. The other event's docs should only link to `Document#change`.